server/server_test.go

```diff
@@ -26,7 +50,7 @@ var _ = Describe("Server", func() {
 	)
 
 	BeforeEach(func() {
-		d, err := gorm.Open("postgres", "postgres://fpfujvlpxoelcm:93ec17a8d9323c29e05b569ea2bd77fa2d7dc96564d2f5b6eaa521807bf8b787@ec2-54-243-128-95.compute-1.amazonaws.com:5432/d34p830c249n99")
+		d, err := gorm.Open("postgres", "postgres://eldbficyapouiq:246af417c764c5c0bbfb23f5e26a69033c45a27c515b81611b3830447b757263@ec2-107-20-167-11.compute-1.amazonaws.com:5432/dfgl7a5728belb")
```

Comment:
It would be better to load this from environment using `godotenv` package.

Reply:
@BharathTalakad See the changes!